crux-llvm/test-data/golden/golden/vector-cmp.good

```diff
@@ -1,10 +1,2 @@
-let -- internal
-    v10 = floatFromBinary 0x40a00000:[32]
-    -- internal
-    v11 = floatFromBinary 0x42280000:[32]
-    -- internal
-    v50 = ite (floatLe v11 v10) 0x0:[1] 0x1:[1]
-    -- internal
-    v52 = bvZext 32 (bvAnd (ite (floatLe v10 v11) 0x1:[1] 0x0:[1]) v50)
- in bvOr 0x1:[32] (bvShl v52 0x1:[32])
+0x3:[32]
```

Comment:
As I understand it, crucible/what4 is now able to concretely evaluate both branches of this test case as opposed to just the one case, which is good, so this is a valid test, but do we want to create a todo for making more complicated vector tests that will ensure solver vector verification (it now looks like all the vector tests are concrete results), or is that no longer a viable consideration with the what4 changes?

Reply:
I think the aspects being tested here are really just the LLVM translation of vector values.  If I remember correctly, the vector components are treated as scalars by the What4 layer.  I don't think we need a separate test.